README.md

```diff
@@ -50,18 +51,7 @@ tracksService.get(trackIds).flatMap { tracks =>
 ```
 
 The same composition using Clump:
-
-```scala
-Clump.traverse(trackIds) { trackId =>
-  trackSource.get(trackId).flatMap { track =>
-    userSource.get(track.creator).map { user =>
-      new EnrichedTrack(track, user)
-    }
-  }
-}
-```
-
-Or expressed more elegantly with a for-comprehension:
+(also makes just 1 call to tracksService and 1 call to usersService)
```

Comment:
Could you improve the format here? It doesn't look well with gihub's md presentation.